### PR TITLE
[17.0][MIG] account_invoice_warn_message

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
Code review :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

File: account_invoice_warn_message/models/account_move.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    invoice_warn_msg = fields.Text(compute="_compute_invoice_warn_msg")
+
+    @api.depends(
+        "move_type",
+        "state",
+        "partner_id.invoice_warn",
+        "partner_id.parent_id.invoice_warn",
```
Comment (JordiMForgeFlow) [inline account_invoice_warn_message/models/account_move.py:16]: not sure if maybe it would be better to use partner_id.commercial_partner_id?

**Pull request merged**